core/java/iesi-core/src/main/java/io/metadew/iesi/datatypes/dataset/implementation/inmemory/InMemoryDatasetImplementationJsonComponent.java

```diff
@@ -1,18 +0,0 @@
-package io.metadew.iesi.datatypes.dataset.implementation.inmemory;
```

Comment:
remove file

Reply:
As you can see, this file is already deleted.